CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.10.0 – 2023-05-02
+### Changed
+- Allow to disable users instead of deleting them (#31)
+- Add UI setting for `keep_users_without_login` (#118)
+- Compatibility with Nextcloud 27
```

Comment:
I dislike versions claiming to be compatible with "upcoming" Nextcloud versions. There could always be an breaking change last minute :P (but that might be personal preference).

Reply:
Well, then we should not have merged #522, no? :wink: